conduct.md

```diff
@@ -14,7 +14,32 @@
 * Participation
 * Decisions
 * Postmortems
-* Conflicts
+
+## Conflicts
+
+It is inevitable that conflicts will arise when working with a large group of
+people. It is essential, however, to resolve these conflict in a professional
+manner while keeping in mind the concerns of the people involved.
+
+* **Acceptable Conduct (A):**
+
+  * *Members shall*
+
+    * Resolve conflicts before they become severe by engaging in respectful
+    conversations
+    * If conflicts become severe within a group of people it should be brought
+    to the attention of the project managers who will work to resolve any issues
+    * Be respectful and open minded with everybody in the classroom to benefit
```

Comment:
open-minded. This sentence is also a little bit wordy too but that's just my opinion.